ehr2vec/dataloader/collate_fn.py

```
@@ -10,10 +10,22 @@ def static(data: list) -> dict:
     return padded_data
 
 
-def dynamic_padding(data: list) -> dict:
+def get_bucket_length(length, buckets=[64, 128, 256, 512, 1024, 2048]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)